lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
> OK it seems to be fine after creating the symlink to /Applications/GAP/gap4r7/pkg/. Here is the output.

Great! The build log is redirected to make_manuals.out, you may also wish to check those files.